pkg/kubelet/eviction/types.go

```diff
@@ -88,7 +88,7 @@ type statsFunc func(pod *api.Pod) (statsapi.PodStats, bool)
 type rankFunc func(pods []*api.Pod, stats statsFunc)
 
 // signalObservations maps a signal to an observed quantity
-type signalObservations map[Signal]resource.Quantity
+type signalObservations map[Signal]*resource.Quantity
```

Comment:
this changed for convenience?  i assume there was no issue using quantity in a map value.